__tests__/integration.test.js

```diff
@@ -221,7 +240,7 @@ describe.each(clients)('%s', (_, client) => {
         } else {
           // It doesn't matter that the /right/ boundary is set up because some targets may add their own, we just
           // need to make sure that **a** boundary is present.
-          const contentTypes = expected.headers['Content-Type'].split(';').map(p => p.trim());
+          const contentTypes = response.headers['Content-Type'].split(';').map(p => p.trim());
```

Comment:
This typo was causing a false positive in the `multipart-form-data` suite test by never asserting that we were actually getting `multipart/form-data` back in the response.